src/main/steps/common/switch.to.sole.content.utils.test.ts

```diff
@@ -0,0 +1,130 @@
+import dayjs from 'dayjs';
```

Comment:
I think the name should be src/main/steps/common/switch-to-sole-content.utils.test.ts. The purpose of the dots is to separate the purpose of the file i.e. a utils file or a tests file.

Reply:
Thanks - will resolve in the next commit.